vendor/engines/saml_authentication/lib/saml_authentication/engine.rb

```diff
@@ -14,6 +14,8 @@ class Engine < Rails::Engine
       ViewHook.add_hook "devise.sessions.new",
                         "before_login_form",
                         "saml_authentication/sessions/new"
+
+      OneLogin::RubySaml::Logging.logger.level = Logger::DEBUG
```

Comment:
I assume we won't want this here once this PR is no longer WIP.

Reply:
Yeah, we won't, but it's been super helpful in development.